docs/content/latest/yugabyte-cloud/cloud-connect/add-users.md

```diff
@@ -13,15 +13,15 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Create database users to provide clients access to the cluster's YugabyteDB database. A database user's access is determined by the roles assigned to the user. YugabyteDB uses role-based access control (RBAC) to manage authorization.
+Create database users to provide clients access to the cluster's YugabyteDB database. A database user's access is determined by the roles assigned to the user. YugabyteDB uses [role-based access control](../../cloud-security/cloud-users/) (RBAC) to manage authorization.
```

Comment:
The link you are pointing to is taking us to the **Database authorization in Yugabyte Cloud clusters** page instead of the role-based access control page. Shouldn't you be linking like this : YugabyteDB uses role-based access control (RBAC) to [manage authorization](../../cloud-security/cloud-users/).